lib/Field/RepeatableSettings.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { bool, number } from 'prop-types';
+import { ButtonIcon } from '../ButtonNew/ButtonIcon/ButtonIcon';
+import { ButtonIconDanger } from '../ButtonNew/ButtonIcon/ButtonIconDanger';
+
+const RepeatableSettings = ({
+  isTextDirLTR,
+  fieldIsActive,
+  repeatPosition,
+  isLastRepeat
+}) => {
+  const label = `${repeatPosition < 9 ? '0' : ''}${repeatPosition + 1}`;
+
+  return (
+    <div
+      className={`
+      text-neutral-primary
+      text-lg
+      leading-snug
+      flex
+      items-center
+      -my-3
+      ${isTextDirLTR ? 'ml-auto' : 'mr-auto'}
+    `}
+    >
+      <div
+        className={`
+        ${fieldIsActive ? '' : 'opacity-0'}
+        group-hover:opacity-100
```

Comment:
This pattern means that if the prop `fieldIsActive` is true (e.g. when editing content in the field), it will just have `opacity: inherit` (i.e. fully visible).
The group-hover trick means that anytime someone hovers over the parent Field (with class `.group`) that these styles will be applied, so even if a field isn't active, if you hover anywhere over the field the opacity will animate to 100%. Pretty neat!